src/pseudopeople/utilities.py

```diff
@@ -155,7 +155,7 @@ def two_d_array_choice(
     options["choice_index"] = choice_index
     idx, cols = pd.factorize(options["choice_index"])
     # 2D array lookup to make an array for the series value
-    new = pd.Series(
+    new: pd.Series = pd.Series(
```

Comment:
This is very silly to me that I had to type hint this since I'm literally invoking `pd.Series(...)` 🤷‍♂️